solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -526,8 +527,35 @@ describe("audius-data", function () {
       adminStgPublicKey: adminStgKeypair.publicKey,
     });
 
-    // New sol key that will be used as user authority delegate
+    // Init AuthorityDelegationStatus for a new authority
     const userAuthorityDelegateKeypair = anchor.web3.Keypair.generate();
+    const authorityDelegationStatusSeeds = [
+      Buffer.from("authority-delegation-status", "utf8"),
+      userAuthorityDelegateKeypair.publicKey.toBytes().slice(0, 32),
+    ];
+
+    const authorityDelegationStatusRes = await PublicKey.findProgramAddress(
+      authorityDelegationStatusSeeds,
+      program.programId
+    );
+    const authorityDelegationStatusPda = authorityDelegationStatusRes[0];
+
+    const initAuthorityDelegationStatusArgs = {
+      accounts: {
+        delegateAuthority: userAuthorityDelegateKeypair.publicKey,
+        authorityDelegationStatusPda: authorityDelegationStatusPda,
+        payer: provider.wallet.publicKey,
+        systemProgram: SystemProgram.programId,
+      },
+      signers: [userAuthorityDelegateKeypair],
```

Comment:
nah the signer should be the only one needed to initialize itself as a delegate

Reply:
but the user has to OK the auth delegation for the auth to be grantable, right?

Reply:
this only has to do with one user and that is the delegate. only the delegate has to sign and say it can become a delegate. it doesn't say who it will become a delegate for. that would be later on add_user_authority_delegate that establishes this relationship.